tools/triage/add_triage_signature.py

```diff
@@ -134,6 +134,17 @@ def get_metadata_json(cluster_url):
         raise FailedToGetMetadataException from e
 
 
+def get_manifests(cluster_url):
+    try:
+        res = requests.get("{}/cluster_files/manifests".format(cluster_url))
```

Comment:
any reason why we choose format over f-strings?

Reply:
hmm just copied the metadata function